### PR TITLE
(SVACE) fix major issues (part 2) 

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
Contains the following commits:
 * BaseKeyboardSupport.js (SVACE): early return in case of wrong direction
 * Checkboxradio.js (SVACE): do not always return the same value
 * DropdownMenu.js (SVACE): add return to _configure
 * BaseKeyboardSupport.js (SVACE): add return to getDistanceByDirection
 * Drawer.js (SVACE): check right position
 * Dimmer.js (SVACE): add return to _refreshValue
 * BaseKeyboardSupport.js (SVACE): remove return from _onShortPress

### CODE REVIEW

File: src/js/core/widget/BaseKeyboardSupport.js [hunk 1]
```diff
@@ -1187,7 +1187,7 @@
 				var self = this;
 
 				if (!ns.getConfig("keyboardSupport", false)) {
```
Comment (TomaszLukawskiSam) [inline src/js/core/widget/BaseKeyboardSupport.js]: I wondering about inversion of this condition and move the code below into this conditional.

Comment (TomaszLukawskiSam) [reply src/js/core/widget/BaseKeyboardSupport.js]: I cannot verify this patch is solving SVACE issue. How you verify it?

Comment (grzegorz-czajkowski) [reply src/js/core/widget/BaseKeyboardSupport.js]: I don't know the way to check if this issue is solved or not. Usually next run tells us if we achieved required number of svace issues.
I just tested UIComponets app if there is no regression after this change.

Comment (grzegorz-czajkowski) [reply src/js/core/widget/BaseKeyboardSupport.js]: Both options are fine to me. I can change the code according to your comment.

Comment (TomaszLukawskiSam) [reply src/js/core/widget/BaseKeyboardSupport.js]: I prefer do inversion of condition and move the code into the condition block.

I reviewed all commit in this PR. I think this is only thing to fix.

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

**Comment by TomaszLukawskiSam** [comment]
I installed SonarLint in my VSC and enabled several rules enabled also in SVACE:
```
{
    "sonarlint.rules": {
        "javascript:S3801": {
            "level": "on"
        },
        "javascript:S3271": {
            "level": "on"
        },
        "javascript:S3758": {
            "level": "on"
        }
    }
}
```

obraz

File: src/js/core/widget/BaseKeyboardSupport.js (hunk 1 above)
Comment (grzegorz-czajkowski) [reply src/js/core/widget/BaseKeyboardSupport.js]: Done.

**Review by TomaszLukawskiSam — APPROVED** [review]

**Pull request merged**